### ISSUE TITLE
Support inlined sources

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode-chrome-debug-core#85]
If local sources can't be resolved, use the sources inlined in the sourcemap

**Comment by auchenberg** [comment]
How does the inline source maps show up inside the editor? As a "virtual file" like in Chrome DevTools?

**Comment by roblourens** [comment]
It doesn't show up in the file explorer, you can only see the source if you step in to it, or have a debugger keyword, or something. But once you see the source, you can set more breakpoints in it.

**Comment by roblourens** [comment]
Relevant - https://github.com/Microsoft/vscode/issues/11947

**Comment by auchenberg** [comment]
Got it, so as virtualized file. Could be interesting to do some fuzzy match on the file-contents, and map it that way as a last resort.

**Comment by spencerhakim** [comment]
I'm having trouble with this using `vscode-node-debug2` and `babel-register`. Using breakpoints or a `debugger;` statement both result in the same behavior: my original source file is displayed (from the file system, not as a virtual file), but the debugger points at the wrong lines as I step through it, to the point where it thinks I've reached the end of my test file before it's even executed the first `console.log`. I've verified that base64 inline maps are generated.

Not sure if this is the right issue or project to report this, but it seemed like the most relevant.
